docs/specification/attributes.md

```diff
@@ -34,6 +34,7 @@ Attributes are annotated with the `origin` of the data.  This field can have the
 
 - `measured`: The Value of this Attribute was directly measured.
 - `predicted`: The Value of this Attribute came from a model, such as a complex simulation, a machine learning--derived computation or rule-of-thumb estimation
+- `summary`: The Value of this Attribute is an aggregate of multiple values.
```

Comment:
```suggestion
- `summary`: The Value of this Attribute is calculated based on a set of the same property at a finer level of granularity (aggregation of data).
```
Exactly coping the text from:
https://citrineinformatics.github.io/gemd-docs/high-level-overview/#attribute-in-specs-and-runs

Possibly, we should copy the other way instead.

Reply:
I agree it should be the same text in both places. I do not have a strong opinion as to the specific text.